saleor/core/utils/random_data.py

```diff
@@ -203,9 +205,19 @@ def create_product_variants(variants_data):
             continue
         defaults['product_id'] = product_id
         defaults['attributes'] = json.loads(defaults['attributes'])
+        defaults['price_override'] = get_in_default_currency(
+            defaults, 'price_override', settings.DEFAULT_CURRENCY)
+        defaults['cost_price'] = get_in_default_currency(
+            defaults, 'cost_price', settings.DEFAULT_CURRENCY)
         ProductVariant.objects.update_or_create(pk=pk, defaults=defaults)
 
 
+def get_in_default_currency(defaults, field, currency):
+    if dict.get(defaults, field):
```

Comment:
It's rather uncommon in Python to call the dict methods like that. Instead, you could simply write:
```
if field in dict:
```